### ISSUE TITLE
Unable to run sample

### ISSUE THREAD

**Issue by valhuber** [goodking-bq/graphene-sqlalchemy-auto#2]
Very interesting project - would love to discuss.  I am trying to integrate it it with [Logic Bank](https://github.com/valhuber/logicbank#readme)

To run example/run.py, I had to

```
virtualenv venv
source venv/bin/activate
pip install -r requirements.txt
```

and then fix the requirements.txt:
```
graphene-sqlalchemy
flask_aqlalchemy
flask_graphql
```

The read query worked once I removed the filter.

The update query did not work.

**Comment by goodking-bq** [comment]
> Very interesting project - would love to discuss. I am trying to integrate it it with [Logic Bank](https://github.com/valhuber/logicbank#readme)
> 
> To run example/run.py, I had to
> 
> ```
> virtualenv venv
> source venv/bin/activate
> pip install -r requirements.txt
> ```
> 
> and then fix the requirements.txt:
> 
> ```
> graphene-sqlalchemy
> flask_aqlalchemy
> flask_graphql
> ```
> 
> The read query worked once I removed the filter.
> 
> The update query did not work.

you should run example in flask_app or fastapi_app ,and how do update?  
thanks to remind the [Logic Bank](https://github.com/valhuber/logicbank#readme),it's a good project .i well try it.

**Comment by valhuber** [comment]
Will try that... I am valjhuber@gmail.com if you'd like to connect (valhuber on Skype)

**Comment by valhuber** [comment]
OK, I wanted to verify, but I think I was running that app, as follows:
```
cd graphene-sqlalchemy-auto/example/flask_app
export FLASK_APP=app
flask run
```

This is on a Mac, Python 3.8.6

**Comment by valhuber** [comment]
Still unable to run this app, remaining hopeful to attempt Logic Bank integration, but need some help here...

**Comment by valhuber** [comment]
Just coming back to this... happy to see your fix, and anxious to try it with LogicBank.  Have you tried that yet?  

And, significantly, we can now generate a complete server - checkout [ApiLogicServer](https://github.com/valhuber/ApiLogicServer#readme).  Would love to do that for GraphQL...

**Comment by valhuber** [comment]
Hi!

Thanks for the update, but I am not able to run it - I get this:


<img width="1382" alt="graphene-sa-auto-fails" src="https://user-images.githubusercontent.com/5579631/113593204-fd54a180-95ea-11eb-9301-3006e2d9aeb2.png">

I was able to run your prior version except for update.  To verify my understanding, I tried to recreate your project using my database, plus integrating LogicBank into a project called [payment_allocation](https://github.com/valhuber/payment_allocation_graphene#readme).  

It is failing as described in the readme.  I am looking to collaborate to resolve this, if you are interested...
